src/psyclone/tests/domain/gocean/transformations/globalstoargs_test.py

```diff
@@ -91,10 +93,9 @@ def test_globalstoargumentstrans(monkeypatch):
 
     # Monckeypatch resolve_deferred to avoid module searching and importing
```

Comment:
It's a small thing but you've consistently spelt monkeypatch as "monckeypatch" in this file :-) Please could you fix.

Reply:
Ups.. fixed